api/app/src/routes/middlewares/auth.ts

```diff
@@ -41,6 +44,11 @@ export const checkMAPIAccess = async (
   if (hasMAPIAccess) {
     next();
   } else {
+    queryAnalytics({
+      message: "no mapi access",
```

Comment:
Does posthog allows multi-dimensional analytics? I'm wondering if we could have something that enables us to get info about access failures as a whole and, if wanted, we could also go down a level and get info about access failures to MAPI.

Maybe with tags, labels?

NABD though, just wondering if we need to choose one dimension for the event (in this case `no mapi access`) or whether we could have multiple dimensions (something like `["no access", "mapi"]`).

Reply:
We can have multiple dimensions but in the form of properties set.